artifactory.py

```diff
@@ -1313,6 +1313,23 @@ def parent(self):
         obj.timeout = self.timeout
         return obj
 
+    @property
+    def replication_status(self):
+        """
+        Get status of the repo replication
+        :return:
+            namedtuple('status' (bool): True only if OK, 'details' (dict): full response
+        """
+        replication_url = self.drive + "/api/replication/" + self.repo
+        replication_obj = self.joinpath(replication_url)
+        resp = self._accessor.get_response(replication_obj).json()
+
+        success = True if resp["status"] == "ok" else False
+        Replication = collections.namedtuple("Replication", ["status", "details"])
+        replication = Replication(success, resp)
```

Comment:
Why do we need to wrap dict this way?
It gives small advantages, but also hides an information about Artifactory statuses. 

It's easy to create a function in the code (not in the library) that uses the lib
```python
def is_status_ok(data):
   return data['status'] == 'ok'
```

Because we'll be able use it in the `repositories` list too.
![image](https://user-images.githubusercontent.com/4376814/116646888-6c7fa480-a9a3-11eb-9abd-39c50a51c31e.png)

Reply:
in most of cases user will need global replication status, to know if you are not in the middle of the process of replication.

for all other cases you have `replication.details`, from which you can extract all required data

Reply:
ok, please check now